indico_zodbimport/modules/event_regforms.py

```diff
@@ -151,6 +153,9 @@ def _migrate_settings(self):
         self.regform.manager_notification_recipients = sorted(set(old_rf.notification._ccList) |
                                                               set(old_rf.notification._toList))
         self.regform.manager_notifications_enabled = bool(self.regform.manager_notification_recipients)
+        if (layout_settings.get(self.event, 'use_custom_menu')
```

Comment:
wrong logic: if the custom menu is disabled, the registrants menu entry is always enabled. the only case where it's not enabled is when customization is enabled but the menu entry is not enabled.